packages/core/src/actors.ts

```diff
@@ -274,48 +158,46 @@ export function createDeferredBehavior<TEvent extends EventObject>(
 
       return undefined;
     },
+    subscribe: (observer) => {
```

Comment:
"stop handler" in here (a couple of lines above) should probably clear observers